test/api-test.js

```diff
@@ -202,6 +201,11 @@ describe('RippleAPI', function() {
       _.partial(checkResult, responses.getBalances, 'getBalances'));
   });
 
+  it('getBalanceSheet', function() {
+    return this.api.getBalanceSheet(address).then(
+      _.partial(checkResult, responses.getBalanceSheet, undefined));
```

Comment:
Can hook up the response schema here by replacing `undefined` with the schema name. Then it will check that the response always matches the expected schema, testing the code against the schema and the schema against the code.